### PR TITLE
update website with v2.0

### PR DESCRIPTION
**Pull request by kalaninja**
@ahmetalpbalkan please take a look and merge if looks good. I've updated examples and some text. I've decided not to delete the About section, because I need some pr, as it might be my chance to leave this goddamn country )

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,226 +1,220 @@
...
-                <br/>Written in pure Go with no dependencies, tested with 100% code coverage.
+                <iframe src="http://ghbtns.com/github-btn.html?user=ahmetalpbalkan&repo=go-linq&type=watch&count=true&size=large" allowtransparency="true"
+                    frameborder="0" scrolling="0" width="142px" height="30px" style='margin-left:30px'></iframe>
             </p>
 
-            <div class='promo'>
-                <p><iframe src="http://ghbtns.com/github-btn.html?user=ahmetalpbalkan&repo=go-linq&type=watch&count=true&size=large" allowtransparency="true" frameborder="0" scrolling="0" width="142px" height="30px" style='margin-left:30px'></iframe></p>
+            <p>
+                <a class='btn btn-default btn-lg' href='https://github.com/ahmetalpbalkan/go-linq'>Fork it!</a>
+                <a class='btn btn-default btn-lg' href='https://github.com/ahmetalpbalkan/go-linq/wiki'>Wiki</a>
```
Comment (ahmetalpbalkan) [inline index.html]: probably remove wiki?

File: index.html [hunk 2]
```diff
@@ -1,226 +1,220 @@
...
-                            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;return s.(*Student).age >= 20, nil<br/>
+                </tr>
+
+                <tr>
+                    <td class='docs'>
+                        Find those over 20.<br/> So we make type assertion to our input type.
+                    </td>
+
+                    <td class='code'><code>
+                            &nbsp;&nbsp;&nbsp;&nbsp;Where(func (s interface{}) bool {<br/>
```
Comment (ahmetalpbalkan) [inline index.html]: you have extra 4 spaces indentation in this `<td>`

File: index.html [hunk 3]
```diff
@@ -1,226 +1,220 @@
...
+                <tr>
+                    <td class='docs'>
+                        Select their ages.
+                    </td>
 
-                        <td class='code'><code>
-                            &nbsp;&nbsp;&nbsp;&nbsp;Select(func (s T) (T, error){<br/>
-                            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;return s.(*Student).age, nil<br/>
+                    <td class='code'><code>
+                            &nbsp;&nbsp;&nbsp;&nbsp;Select(func (s interface{}) interface{} {<br/>
```
Comment (ahmetalpbalkan) [inline index.html]: you have extra 4 spaces indentation in this `<td>` too

File: index.html [hunk 4]
```diff
@@ -1,226 +1,220 @@
...
-                            &nbsp;&nbsp;&nbsp;&nbsp;}).
+                </tr>
+
+                <tr>
+                    <td class='docs'>
+                        Convert domain names to full URLs.
+                    </td>
+
+                    <td class='code'><code>
+                            &nbsp;&nbsp;&nbsp;&nbsp;go func(domains []string) {<br/>
```
Comment (ahmetalpbalkan) [inline index.html]: you have extra 4 spaces indentation in this `<td>` as well, same as below.

**Comment by ahmetalpbalkan** [comment]
left some comments, otherwise good to go! thanks.

**Comment by kalaninja** [comment]
Didn't notice that. I'll fix.

**Pull request merged**